assets/css/v2/pre_fare/flex/one_large.scss

```diff
@@ -0,0 +1,12 @@
+.flex-one-large {
+  position: relative;
+  z-index: 2;
```

Comment:
We might not need `z-index` values since the pre-fare flex zone doesn't involve the "sunken tray" situation that the bus shelter one uses, but we can come back to this during final review.
(I had to set `z-index`es to have drop-shadows display correctly from both the containing tray and the cards "floating" inside the tray)

Reply:
I removed them and watched them for a bit and saw no change. So I think you are right, we shouldn't need them.